internal/testutil/keys.go

```diff
@@ -21,21 +21,21 @@ import (
 	"github.com/mailchain/mailchain/crypto/secp256k1"
 )
 
-var SofiaPrivateKey crypto.PrivateKey     // nolint: gochecknoglobals
-var SofiaPublicKey crypto.PublicKey       // nolint: gochecknoglobals
-var CharlottePrivateKey crypto.PrivateKey // nolint: gochecknoglobals
-var CharlottePublicKey crypto.PublicKey   // nolint: gochecknoglobals
+var SofiaPrivateKey crypto.PrivateKey     //nolint: gochecknoglobals
+var SofiaPublicKey crypto.PublicKey       //nolint: gochecknoglobals
+var CharlottePrivateKey crypto.PrivateKey //nolint: gochecknoglobals
+var CharlottePublicKey crypto.PublicKey   //nolint: gochecknoglobals
 
-// nolint: gochecknoinits
+//nolint: gochecknoinits
 func init() {
 	var err error
-	SofiaPrivateKey, err = secp256k1.PrivateKeyFromHex("01901E63389EF02EAA7C5782E08B40D98FAEF835F28BD144EECF5614A415943F")
+	SofiaPrivateKey, err = secp256k1.PrivateKeyFromBytes("01901E63389EF02EAA7C5782E08B40D98FAEF835F28BD144EECF5614A415943F")
 	if err != nil {
 		log.Fatal(err)
 	}
 	SofiaPublicKey = SofiaPrivateKey.PublicKey()
 
-	CharlottePrivateKey, err = secp256k1.PrivateKeyFromHex("DF4BA9F6106AD2846472F759476535E55C5805D8337DF5A11C3B139F438B98B3")
+	CharlottePrivateKey, err = secp256k1.PrivateKeyFromBytes("DF4BA9F6106AD2846472F759476535E55C5805D8337DF5A11C3B139F438B98B3")
```

Comment:
`secp256k1.PrivateKeyFromBytes` is expecting `[]byte` You need to first do a `hex.DecodeString("DF4BA9F6106AD2846472F759476535E55C5805D8337DF5A11C3B139F438B98B3")` check the error then pass it to  `secp256k1.PrivateKeyFromBytes`.